app/components/product/alias-form.hbs

```diff
@@ -0,0 +1,32 @@
+<ul>
+  {{#each @aliases as |alias|}}
+    <li>
+    <div class="flex">
+      <div class="text-sm my-2 text-gray-900">{{alias}}</div>
+      <Rlv::Button
+        @icon="delete-bin-line"
+        @skin="tertiary"
```

Comment:
I think `@skin="naked" @alert={{true}}` gives a better result as styling for the button